### PR TITLE
Tool 1126 swiftpm

### PR DESCRIPTION
**Pull request by lpusok**

### CODE REVIEW

File: step.yml [hunk 1]
```diff
@@ -286,6 +286,17 @@ inputs:
       value_options:
       - "yes"
       - "no"
+  - cache_level: swift_packages
+    opts:
+      category: Debug
```
Comment (BirmacherAkos) [inline step.yml]: Why is it in the `Debug` category?

Comment (lpusok) [reply step.yml]: Which category would be ok? Create a cache one?

File: main.go [hunk 2]
```diff
@@ -355,37 +364,29 @@ func main() {
...
 	}
 
-	if outputTool == "xcpretty" {
-		xcprettyCmd := xcpretty.New(archiveCmd)
-
-		logWithTimestamp(colorstring.Green, "$ %s", xcprettyCmd.PrintableCmd())
-		fmt.Println()
+	var swiftPackagesPath string
+	if xcodeMajorVersion >= 11 {
+		var err error
```
[the remark this answers is not in the document]
Comment (lpusok) [reply main.go]: It is used in the line `rawXcodebuildOut, err := runArchiveCommandWithRetry(archiveCmd, outputTool == "xcpretty", swiftPackagesPath)`

File: step.yml [hunk 3]
```diff
@@ -74,6 +74,7 @@ inputs:
       - "ad-hoc"
```
Comment (BirmacherAkos) [inline step.yml]: The go dependency install is still there. Could you please remove it?
https://github.com/bitrise-steplib/steps-xcode-archive/blob/d21f1bc5af266f53690dd7d6e3958af487c18d6b/step.yml#L28

File: main.go (hunk 2 above)
[the remark this answers is not in the document]
Comment (lszucs) [reply main.go]: As discussed offline, I'm okay with this construct in the end.

**Comment by lpusok** [comment]
@trapacska Please merge the PR.

**Pull request merged**